dask_planner/src/sql/logical/limit.rs

```diff
@@ -0,0 +1,32 @@
+use crate::expression::PyExpr;
+
+use datafusion::scalar::ScalarValue;
+use pyo3::prelude::*;
+
+use datafusion::logical_expr::{logical_plan::Limit, Expr, LogicalPlan};
+
+#[pyclass(name = "Limit", module = "dask_planner", subclass)]
+#[derive(Clone)]
+pub struct PyLimit {
+    limit: Limit,
+}
+
+#[pymethods]
+impl PyLimit {
+    #[pyo3(name = "getLimitN")]
+    pub fn limit_n(&self) -> PyResult<PyExpr> {
+        Ok(PyExpr::from(
+            Expr::Literal(ScalarValue::UInt64(Some(self.limit.n.try_into().unwrap()))),
+            Some(self.limit.input.clone()),
+        ))
+    }
+}
+
+impl From<LogicalPlan> for PyLimit {
```

Comment:
Should we consider using `TryFrom` instead of `From` since this is fallible?

Reply:
I put up a PR to implement this - https://github.com/dask-contrib/dask-sql/pull/543